ensemble/ctf/window_function_component.py

```diff
@@ -39,14 +65,39 @@ def values(self):
 class WindowOpacityNode(OpacityNode):
     """ An `OpacityNode` with a non-zero radius and a trapezoidal shape.
     """
+    # The function used to generate the opacity curve.
+    window_func = Property(Callable, depends_on=['window_type'])
+
+    # The name of the window function
+    window_type = Enum(DEFAULT_WINDOW_TYPE, values='_window_types')
+    _window_types = List(WINDOW_FUNCTIONS.keys())
```

Comment:
As far as I know, it is if you want to give a default value and a  list of possible values to an `Enum` trait. I'm open to suggestions of course.